src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -35,6 +35,7 @@ export const MapRoom: React.FC<MapRoomProps> = ({
   const isUnclickable = room.type === RoomType.unclickable;
   const isMapFrame = room.type === RoomType.mapFrame;
   const isCovertRoom = room.type && COVERT_ROOM_TYPES.includes(room.type);
+  const isLabeled = room.isLabeled !== undefined ? room.isLabeled : true;
```

Comment:
This will use the value if it exists, or default it to `true` if it's `undefined`:
```suggestion
  const isLabeled = room.isLabeled ?? true;
```

**Further Reading:**

- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1632/commits/9e679604114d01a3121e1d3d42ac06b60632cc4a